docs/developerguide.rst

```diff
@@ -56,3 +56,135 @@ We are using Prettier to format our code. In order to use it before a Pull Reque
    .. code-block:: console
 
       $ prettier --write **/*.js
+
+CERN
+----
+
+Prerequisites
+~~~~~~~~~~~~~
+
+You must run the Minikube cluster with VirtualBox VM so you listen requests from outside the
+VM. In order to configure the port forwading, go to the VirtualBox UI and select
+the running VM, then in the *Network* section select *Advanced* > *Port Forwarding*
+
+.. image:: _static/virtualbox-port-forwarding.png
+
+Add a new entry (in the screenshot ``reana-backend``) to forward all the requests going to the port
+443 (default for HTTPS) to the internal port 30443 (where REANA-Server is listening).
+
+Authentication
+~~~~~~~~~~~~~~
+REANA-UI uses Single Sign-On (SSO) for Authentication.
+
+1. `Register <https://sso-management.web.cern.ch/OAuth/RegisterOAuthClient.aspx>`_ an OAuth client.
+
+  * ``client_id``: A unique text string that identifies your client, e.g. ``my-reana-ui``.
```

Comment:
Instead of `my-reana-ui` and `my-reana` we could use everywhere the same `reana-johndoe` (in synchrony with the naming scheme we use for production instances `reana[-qa|-dev]`) which you are already using [here](https://github.com/reanahub/reana-ui/pull/40/files#diff-2b2f3fdab1b53ff84ce39f48ddf75281R115).